eris.go

```diff
@@ -446,6 +454,14 @@ func (e *rootError) Is(target error) bool {
 
 // As returns true if the error message in the target error is equivalent to the error message in the root error.
 func (e *rootError) As(target any) bool {
```

Comment:
```suggestion
// As returns true if the error message in the target error is equivalent to the error message in the root error.
// In case of a joined root error, returns true if the error message in the target error is equivalent to at least one of the error messages in root.
func (e *rootError) As(target any) bool {
```